Client/qtTeamTalk/utiltts.h

```diff
@@ -96,13 +96,10 @@ enum TextToSpeechEngine
     TTSENGINE_NONE = 0,
     TTSENGINE_QT = 1,
     TTSENGINE_TOLK = 2,
-    TTSENGINE_NOTIFY = 3,

```

Comment:
Better to rename `TTSENGINE_NOTIFY` to `TTSENGINE_NOTIFY_OBSOLETE`, so we don't reduce the value `3`.